### ISSUE TITLE
Starred messages don't appear in bundles

### ISSUE THREAD

**Issue by nikonikolov** [teresa-ou/inboxy#59]
If an email is marked with a star and a particular label, when I expand the bundle, the the email does not appear there. The email is only visible in the main inbox.

**Comment by teresa-ou** [comment]
Yes, this is a known issue. The extension only visually reorders the emails to show them in bundles, because it seems that Gmail's javascript expects them to remain in the same order in the html. inboxy is relying on Gmail to open the email when the email is clicked, so I haven't been able to show a starred message in both the bundle and the main message list.

**Comment by CaneRaiser** [comment]
Is it possible for starred messages to behave like pinned messages did in inbox?  Just show up in the bundle, but don't sweep when you mark the bundle as done?

**Comment by benoror** [comment]
Hi! Starred messages staying outside the bundle is intentional in **Inbundly** (and matches old Inbox pin behavior).

Options → Bundle setup → **Keep starred messages outside bundles** (on by default). Turn it off if you want starred threads to remain inside their label bundles.

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Site: https://inbundly.com/
If you saw starred vanish even with the option off, please file a bug: https://github.com/benoror/inbundly/issues/new
